spotinst/azure_v3/elastigroup_azure_vm_sizes/consts.go

```diff
@@ -6,6 +6,10 @@ const (
 	Prefix = "azure_vm_sizes_"
 )
 
+const (
+	VmSizes commons.FieldName = "vm_sizes"
+)
+
```

Comment:
not necessarily to be a separate const, please combine

Reply:
Combined into one const